README.md

```diff
@@ -1,3 +1,15 @@
 # marta-ga-us
 
-Information about MARTA (Atlanta, GA - http://www.itsmarta.com/).
+Tracking issues for [MARTA](http://www.itsmarta.com/) (Atlanta, GA) GTFS and GTFS-realtime data.
```

Comment:
• `DE` could mean Delaware or Germany (aka Deutschland), therefore I would push to say `(Atlanta, GA, US)`. Same thing for `GA` which is Georgia (the US state) or Georgia the caucasian state.

• Maybe we should put somewhere the expended version of marta, aka "Metropolitan Atlanta Rapid Transit Authority", for searching purpose.